articles/openshift/howto-integrate-elasticsearch-operator.md

```diff
@@ -37,20 +37,32 @@ Follow the instructions in these tutorials and then return here to continue.
    > [!NOTE]
    > To specify the name of an existing **StorageClass** for Elasticsearch storage in step **Create a Cluster Logging instance**, open **ARO web console** > **Storage** > **Storage Classes** and find the supported storage class name.
 
-After the newly created Cluster Logging instance is up and running, configure Fluentd to merge the JSON log message bodies emitted by sample application.
+Now configure Fluentd to merge the JSON log message bodies emitted by sample application.
 
 1. Switch to project `openshift-logging`:
 
    ```bash
    oc project openshift-logging
    ```
 
-2. Change the cluster logging instance’s **managementState** field from **Managed** to **Unmanaged**:
+2. Change the cluster logging instance’s **managementState** field from **Managed** to **Unmanaged** using `oc edit` command:
 
    ```bash
    oc edit ClusterLogging instance
+
+   apiVersion: "logging.openshift.io/v1"
+   kind: "ClusterLogging"
+   metadata:
+     name: "instance"
+
+   ....
+
+   spec:
+     managementState: "Unmanaged"
    ```
 
+   The `oc edit` command is equivalent to `kubectl edit` command. For more information, please see [kubectl edit](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands/#edit) documentation.
```

Comment:
One slight worry about only linking to the Kube docs for `edit` is there is at least one small difference (e.g. `OC_EDITOR` to select the editor).  Maybe also link to https://docs.openshift.com/container-platform/4.6/cli_reference/openshift_cli/developer-cli-commands.html#edit

Reply:
Good idea, added reference to the `oc edit` doc with [this commit](https://github.com/majguo/azure-docs/pull/3/commits/9903e088a2e6ee7cc0bf6a23f62fb3d3f441314e)